cluster/gce/util.sh

```diff
@@ -20,6 +20,14 @@
 # config-default.sh.
 readonly GCE_MAX_LOCAL_SSD=8
 
+BASE64_HELP=$(base64 --help)
+BASE64_INPUT_FLAG=""
+if [[ $BASE64_HELP =~ "--input" ]]; then
+  BASE64_INPUT_FLAG="-i"
```

Comment:
let's use `--input` since we found it, instead of assuming it has the `-i` alias.

Reply:
done